h5netcdf/tests/test_h5netcdf.py

```diff
@@ -1689,17 +1689,23 @@ def test_bool_slicing_length_one_dim(tmp_local_netcdf):
 def test_default_chunking(tmp_local_netcdf):
     with h5netcdf.File(tmp_local_netcdf, "w") as ds:
         ds.dimensions = {"x": 10, "y": 10, "z": 10, "t": None}
-        v = ds.create_variable("hello", ("x", "y", "z", "t"), "float", chunks="h5py")
+        v = ds.create_variable(
```

Comment:
Uhm yea this test didn't make any sense, it was just testing the same thing 3 times. Don't know what I was thinking 🤷

Reply:
The new tests should cover all cases, including manually resized dimensions. I don't see much of a point in testing with `data=` since that's only possible after a manual resize, so this case should be covered well.

Reply:
If we need more tests (we'll find out anyway), we can add them later. Thanks.